cmd/vocli/commands/account.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+	"go.vocdoni.io/dvote/client"
+)
+
+var accInfoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "Get information about an account from the vochain. The address may or may not include the 0x prefix",
```

Comment:
Maybe is not really needed to say that the account can be passed with 0x or without. If we want to point this we should also point that the address can be checksumed or not.